src/learning-header/AuthenticatedUserDropdown.jsx

```diff
@@ -4,28 +4,39 @@ import PropTypes from 'prop-types';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faUserCircle } from '@fortawesome/free-solid-svg-icons';
 import { getConfig } from '@edx/frontend-platform';
+import { Avatar } from '@openedx/paragon';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { Dropdown } from '@openedx/paragon';
 
 import messages from './messages';
 
-const AuthenticatedUserDropdown = ({ intl, username }) => {
+const AuthenticatedUserDropdown = ({
+  intl, username, avatar, name,
```

Comment:
Please enlighten me here with the occurrences passing avatar to `AuthenticatedUserDropdown`

Reply:
Please see the screenshot above. The original Learning Header just has the "username" in it. When we hide username, we want to be able to have something in place of it and that is why Mubbshar has passed avatar.

Reply:
I understand the purpose for passing avatar. I was just finding the places where AuthenticatedUserDropDown is rendered and avatar is passed to it.

Reply:
`AuthenticatedUserDropdown` is rendering from the `LearningHeader` but `avatar` is not passing in it. There could be a use case to display the user profile image, but for now we are not allowing the user to upload your profile image.

Reply:
can we pass it? if user has not uploaded the image, it will not be shown. I don't think disabling of user profile images should affect here.

Reply:
Currently, there is no profile image in user's profile, but if we want to show profile image we need to pass it from the `Header` component.
But anyway, we can remove avatar prop from the component.

Reply:
I think I am getting confused here. All I just want is to pass avatar in `AuthenticatedUserDropdown` because we are providing its functionality in component by accepting it as a prop. If the avatar is not available in user profile it will automatically not be shown and if its available it will be shown.

Please help me clear my confusion.

Reply:
We have removed the avatar prop from the component which we added to use in the future if we allow users to upload the image, but initially, in `LearningHeader` component there was no such support for avatar.
I hope it clears up all your confusion.